tfmesos/scheduler.py

```diff
@@ -215,7 +218,7 @@ def resourceOffers(self, driver, offers):
         '''
 
         for offer in offers:
-            if all(task.offered for task in self.tasks):
+            if all(task.offered for id, task in self.tasks.iteritems()):
```

Comment:
Maybe we should use `six.iteritems(self.tasks)` or `self.tasks.values()` to make this compatible for both Python2 and Python3